src/core/sys/posix/sys/types.d

```diff
@@ -247,34 +247,15 @@ else version (CRuntime_Bionic)
     alias c_long    time_t;
     alias uint      uid_t;
 
-    version (X86)
-    {
-        alias ushort    mode_t;
-        alias ushort    nlink_t;
-    }
-    else version (X86_64)
-    {
-        alias ushort    mode_t;
-        alias uint      nlink_t;
-    }
-    else version (ARM)
-    {
-        alias ushort    mode_t;
-        alias ushort    nlink_t;
-    }
-    else version (AArch64)
-    {
-        alias uint      mode_t;
-        alias uint      nlink_t;
-    }
-    else version (MIPS32)
```

Comment:
[Android NDK 17, released earlier this year, dropped support for MIPS](https://github.com/android-ndk/ndk/wiki/Changelog-r17), allowing this simplification. Nobody was using Android/MIPS anyway.

The `x64` `mode_t` declaration above was wrong.